liminal/runners/airflow/model/task.py

```diff
@@ -28,15 +28,14 @@ class Task(ABC):
     """
 
     def __init__(self, task_id, dag, parent, trigger_rule, liminal_config, pipeline_config,
-                 task_config, executor=None):
+                 task_config):
         self.liminal_config = liminal_config
         self.dag = dag
         self.pipeline_config = pipeline_config
         self.task_id = task_id
         self.parent = parent
         self.trigger_rule = trigger_rule
         self.task_config = task_config
-        self.executor = executor
 
     @abstractmethod
     def apply_task_to_dag(self, **kwargs):
```

Comment:
We should aim to have `apply_task_to_dag` only be in executor
We can have tasks that have this implementation implement a super called `AirflowTask` and change `DefaultExecutor` to be `AirflowExecutor` and use the executor to call a method of `AirflowTask` which augments the DAG which tasks like `create_cloudformation_stack` can then implement